sdk/storage/azblob/appendblob/client_test.go

```diff
@@ -1259,7 +1256,7 @@ func (s *AppendBlobRecordedTestsSuite) TestBlobCreateAppendIfMatchTrue() {
 	validateAppendBlobPut(_require, abClient)
 }
 
-func (s *AppendBlobRecordedTestsSuite) TestAppendSetImmutabilityPolicy() {
+func (s *AppendBlobRecordedTestsSuite) immutabilityPolicySetting() {
```

Comment:
Why are we changing this? Shouldn't it have "Test" prefix?

Reply:
It got changes while refactoring the variable name automatically. Fixed it.